src/token/token_sig.ml

```diff
@@ -30,4 +32,8 @@ module type SERVICE = sig
     unit ->
     (Token_core.t option, string) Result.t Lwt.t
   (** Returns an active and non-expired token *)
+
+  val invalidate :
+    Core.Ctx.t -> token:Token_core.t -> unit -> (unit, string) Lwt_result.t
+  (** Invalidates a token, subject to deletion *)
```

Comment:
@aronerben Does this mean the token is subject to deletion?
We still have to agree on documentation best practices but I started to:
* use imperative
* make complete sentences, no abbreviations and ambiguities

Reply:
Done